test/ex_doc/formatter/html_test.exs

```diff
@@ -527,5 +527,17 @@ defmodule ExDoc.Formatter.HTMLTest do
              |> HTML.ast_to_html()
              |> IO.iodata_to_binary() == "<strong><em>bar</em></strong>"
     end
+
+    test "handles empty elements" do
+      assert """
+             <span>
+             <i></i>
+             </span>
+             """
+             |> ExDoc.Markdown.Earmark.to_ast([])
+             |> HTML.ast_to_html()
+             |> IO.iodata_to_binary() ==
+               "<span><i></i></span>"
```

Comment:
This test passes on master without your change, could you rebase and try again?

I was able to see a failure with such markdown though: `<i></i>`, If you can verify it, could you test that instead ?

Reply:
Ah ok, my test case initially contained two `<i></i>` and it failed. After fixing that I decided to remove one of the elements and it still worked but I did not test that on `master`. I will make the test more explicit.